CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 0.2.0 (2016-02-xx)
+
+* Improve the styling on the login and dashboard list pages
+* Use notebook file names as dashboard page titles
+* Fix errors when notebook filenames have spaces
+* Fix support for `%%javascript` in dashboards
+* Support subdirectories on the dashboard list page
+* Enable initial support for declarative widgets in dashboard server
```

Comment:
Will do. I'm going to add a bit more to the README as well before taking off the WIP.